src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo__img {
```

Comment:
The `display: flex;` property here seems unnecessary. The `display: flex;` property is typically used to create a flex container and lay out its children using flex layout. However, in this case, `.logo__img` doesn't appear to have any children. If `.logo__img` is just an `img` tag, you can remove this property.